lua/entities/sent_spawnpoint/init.lua

```diff
@@ -29,64 +34,60 @@ function unlinkPlayerFromSpawnPoint( player, spawnPoint )
 end
 
 function unlinkAllPlayersFromSpawnPoint( spawnPoint, excludedPlayers )
-    if ( IsValid(spawnPoint) ) then
-        local linkedPlayers = spawnPoint.LinkedPlayers
-        local spawnPointOwner = spawnPoint:CPPIGetOwner()
-
-        for player, _ in pairs( linkedPlayers ) do
-            local playerIsExcluded = excludedPlayers[player]
-            local playerOwnsSpawnPoint = spawnPointOwner == player
-			
-            if ( not playerIsExcluded and not playerOwnsSpawnPoint ) then
-                unlinkPlayerFromSpawnPoint( player, spawnPoint )
-                player:PrintMessage(4, "You've been unlinked from a Spawn Point!")
-            end
+    if not IsValid( spawnPoint ) then return end
+
+    local linkedPlayers = spawnPoint.LinkedPlayers
+    local spawnPointOwner = spawnPoint:CPPIGetOwner()
+    
+    for player, _ in pairs( linkedPlayers ) do
+        local playerIsNotExcluded = excludedPlayers[player] == nil
+        local playerIsNotSpawnPointOwner = spawnPointOwner ~= player
```

Comment:
shouldnt the owner sometimes be removed such as in ENT:OnRemove()?
The owner is also in excludedPlayers when it is called from the command `!unlinkthis`.

Reply:
Dude, I have lost *all context* on this.

You'll need to give me at least a couple months to ramp back up on this effort.

Reply:
Seriously, though, if you have the bandwidth it'd be terrific if you fixed whatever you're talking about.

Reply:
@justinmuskopf very irresponsible

Reply:
I think thats how it was before so it shouldnt break anything if its merged.